cmake/Modules/Findfmt.cmake

```diff
@@ -0,0 +1,64 @@
+add_library(fmt INTERFACE IMPORTED)
+
+set(fmt_external_project_name fmtlib_fmt)
```

Comment:
Quite misleading, it's fine to duplicate project name, since it a common practice.